src/tests/ftest/util/test_utils_pool.py

```diff
@@ -554,6 +554,30 @@ def get_pool_daos_space(self):
         keys = ("s_total", "s_free")
         return {key: getattr(self.info.pi_space.ps_space, key) for key in keys}
 
+    def get_pool_free_space(self, device="scm"):
+        """
+        Method Description
+             Get SCM or NVME free space
+        Keyword Arguments:
+            device {str} -- (default: {"scm"} or "nvme")
+
+        Returns:
+            free_space : Free SCM or NVME space "str"
+        """
+        free_space = 0
+        dev = device.lower()
+        if dev == "scm":
+            gindex = 0
+        else:
+            gindex = 1
+        daos_space = self.get_pool_daos_space()
+        for key in sorted(daos_space.keys()):
+            for index, value in enumerate(daos_space[key]):
+                if key == "s_free":
+                    if index == gindex:
```

Comment:
We can use and.
`if key == "s_free" and index == gindex:`

Reply:
Changing the method simple in my next run... No for loops... Based on samir's input. Thanks for the input.